resolwe/toolkit/tests/test_files.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=missing-docstring
+import os
+
+from resolwe.test import ProcessTestCase, with_docker_executor
+
+
+class FilesProcessTestCase(ProcessTestCase):
+
+    def setUp(self):
+        super(FilesProcessTestCase, self).setUp()
+        self.files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), 'files'))
+
+    @with_docker_executor
+    def test_upload_file(self):
+        upload_file = self.run_process('upload-file', {'src': 'file binary'})
+        self.assertFile(upload_file, 'file', 'file binary')
+
+    @with_docker_executor
+    def test_upload_image(self):
+        upload_image = self.run_process('upload-image-file', {'src': 'file image.png'})
+        self.assertFile(upload_image, 'file', 'file image.png')
+
+    @with_docker_executor
+    def test_upload_tabular(self):
```

Comment:
I would break the 4 tests into separate test methods so developers/testers can more easily see progress when calling tests with `-v 2`.